### PR TITLE
Fixes issue #2009

### PR DESCRIPTION
**Pull request by wdwvt1**
Resolves the issue but does not refactor the code to prevent issues like this in the future.

### CODE REVIEW

**Comment by qiime-r2d2** [comment]
Build results will soon be (or already are) available at: http://ci.qiime.org/job/qiime-github-pr/1636/

File: scripts/observation_metadata_correlation.py [hunk 1]
```diff
@@ -199,14 +199,14 @@ def main():
 
     # sort the biom table so that feature values are retrieved in the same
     # order as the metadata in the samples they correspond to
-    bt.sort(sort_f = lambda _: samples_to_correlate, axis='sample')
+    nbt = bt.sort(sort_f = lambda _: samples_to_correlate, axis='sample')
```
Comment (ElDeveloper) [inline scripts/observation_metadata_correlation.py]: Any reason why this should be a separate table than `bt`?

**Comment by ElDeveloper** [comment]
@wdwvt1 can you add a mention in the ChangeLog?

**Comment by gregcaporaso** [comment]
@wdwvt1, could update the ChangeLog with a description of this bug, including when it would happen and what the results would be?

**Comment by gregcaporaso** [comment]
Closing in favor of #2027.

@wdwvt1, I made the requested changes on this so don't worry about it (it was more efficient this way as we're trying to wrap up the release).

**Comment by wdwvt1** [comment]
sorry for the slow reply @gregcaporaso and @ElDeveloper; been tied up with other things. thanks very much for taking care of this.

**Pull request closed without merging**